lib/generator.js

```diff
@@ -642,9 +642,36 @@ class Generator {
     } catch (e) {
       this.templateConfig = {};
     }
+    await this.loadDefaultValues();
```

Comment:
please check some other tests where we check if `validateTemplateConfig` was called, and make sure they are extended to check if `loadDefaultValues` is called

Reply:
Everywhere where you have check for call of `validateTemplateConfig` the `loadTemplateConfig` is mocked. So in previous tests `loadDefaultValues` method is unreachable.

Reply:
🤦 you're right